_layouts/default.html

```diff
@@ -1,13 +1,22 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title> A6 Games | A6 Games</title>
-        {% feed_meta %}
-        {% seo %}
-    </head>
-    <body>
-        <h1>A6 Games</h1>
+
+<head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta charset="utf-8">
+    {% feed_meta %}
+    {% seo %}
+</head>
+<body>
+    {% include header.html %}
```

Comment:
Let's go ahead and add `{{ content }}` underneath this line here.
See [Creating A Layout Section](https://jekyllrb.com/docs/step-by-step/04-layouts/).

This tag won't do anything just yet but it will make sure our default.html will handle loading content from other pages appropriately later.